packages/v4/src/content/design-guidelines/components/label/label.md

```diff
@@ -3,60 +3,75 @@ id: Label
 
 ## Usage
-Labels are easily customized to fit different use cases. There are two main types of labels:
-1. Unfilled
+There are 2 label types:
 
-<img src="./img/unfilled-labels.png" alt="Unfilled labels" width="646"/>
+<img src="./img/label-features-type.png" alt="Different label types" width="180"/>
 
-2. Filled
+1. **Default labels:** Use by default when needing a label.
+2. **Compact labels:** Use when confronted by space limitations and a smaller label is required (for example, in a table).
 
-<img src="./img/filled-labels.png" alt="Filled labels" width="646"/>
 
-Use the filled option to add more visual prominence to a label. If your use case requires a mix of clickable and non-clickable labels, consider using one type for clickable and another for non-clickable. Whatever convention you choose, be sure to maintain consistency through your UI. 
 
-The default colors for labels are gray, blue, green, orange, red, purple, and cyan. We offer a wide array of colors to allow for color coding labels within UIs. If desired, label colors can be used to indicate status. Regardless, we recommend you avoid using the red or green labels unless they indicate danger/error or success, respectively.
+Both default and compact labels come in the following default colors: gray, blue, green, orange, red, purple, and cyan. We offer a wide array of colors to allow for color coding labels within UIs. Label colors can be used to indicate status if desired, but do not have to. Regardless, we recommend you avoid using the red label unless it indicates danger or an error state.
 
-While you may use other colors for your labels, we recommend using the ones provided as they have been carefully selected to be accessible with the PatternFly library. If you choose to use other colors, be mindful of picking accessible text colors and icon colors to go with the label background.
+While you may use other colors for your labels, we recommend using the ones provided as they have been carefully selected to be accessible with the PatternFly library. If you choose to use other colors, be mindful of picking [accessible text color and icon color](https://www.patternfly.org/v4/accessibility/testing-your-accessibility) to go with the label background.
 
 ## Label features
-You can further customize labels by making them dismissable, adding icons, or making them clickable links. You can use any combination of these features in a single label.
+You can further customize labels by making them filled or unfilled, dismissable, adding icons, or making them clickable links. You can use any combination of these features in a single label.
+
 
-<img src="./img/label-features.png" alt="Examples of different types of labels" width="180"/>
+<img src="./img/label-features.png" alt="Examples of different label features" width="180"/>
```

Comment:
Some of these labels have some sizing issues - just fixed them in the design doc assets file if you could just swap out this image for the new one in the sketch file? (the differences are very small but it's the same artboard/file name)

Reply:
This has been applied, thanks.